hwdb/60-evdev.hwdb

```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo ThinkPad T430
```

Comment:
I cannot spot what on earth is wrong with this comment, but `hwdb/parse_hwdb.py` fails with
```
Cannot parse ../hwdb/60-evdev.hwdb: Expected stringEnd (at char 7102), (line:256, col:1)
```

Is there some funny non-standard space there or something similar?